### ISSUE TITLE
error: <identifier> expected if @Value.Style (builder = "new")

### ISSUE THREAD

**Issue by rgrigoryev** [immutables/immutables#89]
I try to make Builder to have non-private constructor. I found that 'new' name disables private constructor generation, but this breaks ``copyOf`` method.
To see compilation error just define class like this:
```
package org.example;

import org.immutables.value.Value;

@Value.Immutable
@Value.Style (builder = "new")
public interface My {
}

```

**Comment by elucash** [comment]
That was a good catch! Logic was corrected to handle this case, pretty regular case. Thank you!
I expect to release it as part of upcoming 2.0, in a week or so. Let me know if you need this fix as a patch for 1.1.x version.

**Comment by rgrigoryev** [comment]
Cool. I look forward to the 2.0.

**Comment by elucash** [comment]
Released in 2.0
